tests/try_scraper/test_invalid_revision.py

```diff
@@ -4,25 +4,16 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
-from mock import patch
-
+import pytest
 from mozdownload import TryScraper
```

Comment:
While `pytest` is a global import, the mozdownload ones are local. As such please separate those blocks with an empty line. Same for the other test.